libraries/plugins/custom_operations/custom_operations.cpp

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2019 oxarbitrage and contributors.
+ *
+ * The MIT License
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+#include <graphene/custom_operations/custom_operations.hpp>
+
+namespace graphene { namespace custom_operations {
+
+void account_storage_map::validate()const
+{
+   FC_ASSERT(key_values.size() <= 10);
```

Comment:
Why this restriction?
Internal storage can be filled over the limit with repeated operations. Single operation pays higher fee for more pairs.

Reply:
removed at https://github.com/bitshares/bitshares-core/pull/1926/commits/2a6cd6166fabb625a36942308a72ea25ba5433b3